eng/pipelines/templates/stages/vmr-build.yml

```
@@ -104,43 +104,45 @@ stages:
     ### Additional jobs for lite/full builds ###
     - ${{ if in(parameters.scope, 'lite', 'full') }}:
 
-      - template: ../jobs/vmr-build.yml
-        parameters:
-          # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-          buildName: ${{ format('{0}_Online_CurrentSourceBuiltSdk', variables.centOSStreamName) }}
-          isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-          vmrBranch: ${{ variables.VmrBranch }}
-          architecture: x64
-          pool: ${{ parameters.pool_Linux }}
-          container: ${{ variables.centOSStreamContainer }}
-          buildFromArchive: false            # 🚫
-          buildSourceOnly: true              # ✅
-          enablePoison: false                # 🚫
-          excludeOmniSharpTests: true        # ✅
-          runOnline: true                    # ✅
-          useMonoRuntime: false              # 🚫
-          withPreviousSDK: false             # 🚫
-          reuseBuildArtifactsFrom:
-          - ${{ format('{0}_Online_MsftSdk_x64', variables.centOSStreamName) }}
-
-      - template: ../jobs/vmr-build.yml
-        parameters:
-          # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-          buildName: ${{ format('{0}_Offline_PreviousSourceBuiltSdk', variables.alpinePreviousName) }}
-          isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-          vmrBranch: ${{ variables.VmrBranch }}
-          architecture: x64
-          artifactsRid: ${{ variables.alpinePreviousX64Rid }}
-          pool: ${{ parameters.pool_Linux }}
-          container: ${{ variables.alpinePreviousContainer }}
-          targetRid: ${{ variables.alpinePreviousX64Rid }}
-          buildFromArchive: false            # 🚫
-          buildSourceOnly: true              # ✅
-          enablePoison: true                 # ✅
-          excludeOmniSharpTests: true        # ✅
-          runOnline: false                   # 🚫
-          useMonoRuntime: false              # 🚫
-          withPreviousSDK: true              # ✅
+      # Disabled until 9.0 GA - see https://github.com/dotnet/source-build/issues/4605
```

This isn't coupled to the GA release.  It is the standing up of net10.0 which is not coupled to the release.

```suggestion
      # Disabled until net9.0 -> net10.0 transition is complete - see https://github.com/dotnet/source-build/issues/4605
```

---

Thanks - I've made a local change, as there are 6 comments, one for each commented-out job. Fixed with https://github.com/dotnet/sdk/pull/43439/commits/9c52eea4bf8ea71ccb2651e46c2f0ba945cd018a